src/objects/Player.java

```diff
@@ -0,0 +1,68 @@
+package objects;
+
+import java.util.Arrays;
+
+public class Player {
+	private Card[] hand;
```

Comment:
You are missing a boolean array to check if the cards are completed or not

Reply:
done.